integration/bundle/bind_resource_test.go

```diff
@@ -58,7 +59,7 @@ func TestBindJobToExistingJob(t *testing.T) {
 	require.Equal(t, job.Settings.Name, fmt.Sprintf("test-job-basic-%s", uniqueId))
 	require.Contains(t, job.Settings.Tasks[0].SparkPythonTask.PythonFile, "hello_world.py")
 
-	c = testcli.NewRunner(t, "bundle", "deployment", "unbind", "foo")
+	c = testcli.NewRunnerWithContext(t, ctx, "bundle", "deployment", "unbind", "foo")
```

Comment:
Should not we always pass context, so WithContext becomes the default mode and NewRunnerWithContext could just become NewRunner?

Reply:
Good point! Renamed in the last commit.